### PR TITLE
refactor(UI): version page changes

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes:

Fixes partially #11221

Version page changes and improvements for pages:

- [x] Table

https://github.com/open-metadata/OpenMetadata/assets/51777795/656c4e5a-6507-40ec-ae28-26df97981472

- [x] Topic

https://github.com/open-metadata/OpenMetadata/assets/51777795/cd719cf6-9eb7-4c9d-9a93-45f9e15d9f48

- [x] Dashboards

https://github.com/open-metadata/OpenMetadata/assets/51777795/6b870a60-8f73-4a8a-b59e-ab4fa2cc0f89

- [x] Pipeline

https://github.com/open-metadata/OpenMetadata/assets/51777795/08234396-a21d-48d2-95f2-cc453af09687

- [x] mlModel

https://github.com/open-metadata/OpenMetadata/assets/51777795/ba61d1b4-4960-4820-98e4-23737901fd14

- [x] Container

https://github.com/open-metadata/OpenMetadata/assets/51777795/8d7d3a71-4818-4f91-b82c-167e7105c184

- [x] Glossary

https://github.com/open-metadata/OpenMetadata/assets/51777795/42400e16-bfac-4ce0-9548-551f95a4502d

- [x] DataModel

https://github.com/open-metadata/OpenMetadata/assets/51777795/06af8fd8-f5f5-473c-a04e-158f6ff257bd

Following are the remaining pages with missing version pages which will be added in a different PR:

- [ ] Service 
- [ ] Database
- [ ] Database Schema 

I worked on updating version pages for different data assets with newly implemented data and bug fixes

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx [hunk 1]
```diff
@@ -295,6 +297,7 @@ const ContainerDataModel: FC<ContainerDataModelProps> = ({
         }}
         pagination={false}
         rowKey="name"
+        scroll={{ x: 1200 }}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx:300]: Let's create a constant for scroll and use it here.

File: openmetadata-ui/src/main/resources/ui/src/components/ContainerVersion/ContainerVersion.component.tsx [hunk 2]
```diff
@@ -395,41 +146,64 @@ const ContainerVersion: React.FC<ContainerVersionProp> = ({
...
-                      containerFQN,
-                      [FqnPart.Column],
-                      FQN_SEPARATOR_CHAR
-                    )}
-                    columns={updatedColumns()}
-                    joins={[]}
+            <Tabs
+              defaultActiveKey={tab ?? EntityTabs.SCHEMA}
+              onChange={handleTabChange}>
+              <Tabs.TabPane
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ContainerVersion/ContainerVersion.component.tsx:164]: We should use items instead of TabPane.

**Review by Sachin-chaurasiya — COMMENTED** [review]
Left a few comments, thanks

**Comment by chirag-madlani** [comment]
@aniketkatkar97 request description/tags / terms are visible in the Version page
We should hide all the actions on the Version page. It should be a read-only experience for the user

**Pull request merged**